src/main/scala/reporting/MultiRunRecorder.scala

```diff
@@ -96,7 +96,7 @@ object MultiRunRecorders extends StatefulComponent {
   private val sinks = mutable.ArrayBuffer.empty[PrintWriter]
 
   protected def config: Config = Verifier.config
-  protected def source: Option[String] = Verifier.inputFile.map(_.toString)
+  var source: Option[String] = None
```

Comment:
Please add a brief description of when/where the value is set to Some(String). I.e. of the implicit usage protocol for the field.